packages/suite/src/support/messages.ts

```diff
@@ -3842,6 +3842,10 @@ export default defineMessages({
         id: 'TR_TOR_DISABLE_ONIONS_ONLY_NO_MORE_DESCRIPTION',
         defaultMessage: 'You can safely disable Tor now.',
     },
+    TR_UNAVAILABLE_COINJOIN_TOR_DISABLE_TOOLTIP: {
+        id: 'TR_TRANSACTIONS_SEARCH_TOOLTIP',
```

Comment:
key and id has to be the same
```suggestion
    TR_UNAVAILABLE_COINJOIN_TOR_DISABLE_TOOLTIP: {
        id: 'TR_UNAVAILABLE_COINJOIN_TOR_DISABLE_TOOLTIP',
```

Reply:
Thank you for caching it! Done https://github.com/trezor/trezor-suite/pull/6796/commits/08f6f4cc786546979a5cf28d50fc402e1281db89